addons/l10n_it_edi_sdi_coop/models/account_edi_format.py

```diff
@@ -0,0 +1,268 @@
+            filename = self._l10n_it_edi_generate_electronic_invoice_filename(invoice)
+            attachment = self.env['ir.attachment'].create({
+                'name': filename,
+                'res_id': invoice.id,
+                'res_model': invoice._name,
+                'datas': base64.encodebytes(xml),
+                'description': _('Italian invoice: %s', invoice.move_type),
+                'type': 'binary',
+            })
+            invoice.l10n_it_edi_attachment_id = attachment
+
+            if len(invoice.commercial_partner_id.l10n_it_pa_index or '') == 6:
+                invoice.message_post(
+                    body=(_("Invoices for PA are not managed by Odoo, you can download the document and send it on your own."))
+                )
+                to_return[invoice] = {'attachment': attachment}
+            else:
+                to_send[filename] = {
+                    'invoice': invoice,
+                    'data': {'filename': filename, 'xml': base64.b64encode(xml)}}
+
+        company = invoice.company_id
+        proxy_user = self._l10n_it_edi_get_proxy_user(company)
+        responses = self._l10n_it_edi_upload([i['data'] for i in to_send.values()], proxy_user, company.l10n_it_edi_test_mode)
+        for filename, response in responses.items():
+            invoice = to_send[filename]['invoice']
+            to_return[invoice] = response
+            if 'id_transaction' in response:
+                invoice.l10n_it_edi_transaction = response['id_transaction']
+                to_return[invoice].update({
+                    'error': _('The invoice was successfully transmited to FatturaPA and is processing'),
+                    'blocking_level': 'info',
+                })
+        return to_return
+
+    def _l10n_it_post_invoices_step_2(self, invoices):
+        ''' Check if the sent invoices have been processed by FatturaPA.
+        '''
+        to_check = {i.l10n_it_edi_transaction: i for i in invoices}
+        to_return = {}
+        company = invoices.company_id
+        proxy_user = self._l10n_it_edi_get_proxy_user(company)
+        responses = proxy_user.make_request('/iap/l10n_it_edi/1/in/TrasmissioneFatture', company.l10n_it_edi_test_mode,
+                                            params={'ids_transaction': list(to_check.keys())})
+        to_ack = []
+        for id_transaction, response in responses.items():
+            invoice = to_check[id_transaction]
+            if 'error' in response:
+                to_return[invoice] = response
+                continue
+
+            state = response['state']
+            if state == 'awaiting_outcome':
+                to_return[invoice] = {
+                    'error': _('The invoice was successfully transmited to FatturaPA and is processing'),
+                    'blocking_level': 'info',
+                }
+                to_ack.append(id_transaction)
```

Comment:
The ack system tells nothing to fatturaPA. It's just to ack to the proxy so it can delete the files. Maybe I can rename the var so it appears more clearly.